.github/workflows/ci.yml

```diff
@@ -7,30 +7,35 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby_version: [2.5.x, 2.6.x, 2.7.x]
-
+        gemfile: [rails_4.0.0.gemfile, rails_5.0.0.gemfile, rails_5.2.3.gemfile, rails_6.0.0.gemfile, rails_master.gemfile]
+        ruby_version: [2.4.x, 2.5.x, 2.6.x, 2.7.x]
```

Comment:
I think 2.4 is no longer maintained.

Reply:
https://www.ruby-lang.org/en/news/2020/04/05/support-of-ruby-2-4-has-ended/

Reply:
we can support ruby 2.4 without much effort and there is 12% of people still using it. https://stats.rubygems.org/

Would you just drop support? Even if rails 4 and rails 5 support ruby 2.4?